File: bricks/pac_native_plugin/__brick__/packages/{{name.snakeCase()}}/android/src/main/kotlin/com/danhdue/{{name.snakeCase()}}/data/{{name.pascalCase()}}DataSource.kt
package com.danhdue.{{name.snakeCase()}}.data

import com.danhdue.{{name.snakeCase()}}.domain.{{name.pascalCase()}}Repository

class {{name.pascalCase()}}DataSource : {{name.pascalCase()}}Repository {
    override suspend fun getStatus(): Result<String> {
        return Result.success("Active")
    }
}

File: bricks/pac_native_plugin/__brick__/packages/{{name.snakeCase()}}/ios/{{name.snakeCase()}}/Sources/{{name.snakeCase()}}/Data/{{name.pascalCase()}}DataSource.swift
// Copyright (c) 2026, one of DanhDue ExOICTIF projects. All rights reserved.

// coverage:ignore-file

import Foundation

public class {{name.pascalCase()}}DataSource: {{name.pascalCase()}}Repository {
    public init() {}

    public func getStatus() async throws -> String {
        return "Active"
    }
}
